src/kirby/components/modal/services/modal-window.helper.tns.ts

```diff
@@ -3,23 +3,27 @@ import { Injectable } from '@angular/core';
 import { ModalDialogService } from 'nativescript-angular/directives/dialogs';
 import { ViewContainerRef } from '@angular/core';
 
-import { ModalComponent } from '../modal.component';
-import { ModalConfig } from '../config/modal-config';
-import { ModalConfigHelper } from '../config/modal-config.helper';
+import { ModalWrapperComponent } from '../modal-wrapper/modal-wrapper.component';
+import { ModalWrapperConfig } from '../modal-wrapper/config/modal-wrapper-config';
+import { ModalWrapperConfigHelper } from '../modal-wrapper/config/modal-wrapper-config.helper';
 
 @Injectable()
-export class ModalHelper {
+export class ModalWindowHelper {
```

Comment:
We chose to loose the `Window` name, so revert back to:
```suggestion
export class ModalHelper {
```

Reply:
yeps, well spotted